shared/src/commonMain/kotlin/com/fbada006/shared/dimensions/Dimensions.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 ArtMaker
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.fbada006.shared.dimensions
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * This is the representation of the Dimensions to be used across all of the platforms.
+ */
+
+object Dimensions {
+    val ColorPickerMenuItemPadding: Dp = 2.dp
+    val ColorPickerMenuItemBorderWidth: Dp = 2.dp
+    val ColorPickerHorizontalArrangement: Dp = 8.dp
+    val ColorItemShapeSize: Dp = 8.dp
+    val CustomColorPickerShapeSize: Dp = 8.dp
+    val StrokeSettingsTopPadding: Dp = 8.dp
+    val StrokeSettingsEndPadding: Dp = 12.dp
+    val StrokeSettingsStartPadding: Dp = 12.dp
+    val ArtMakerControlMenuPadding: Dp = 10.dp
+    val ColorPickerBottomPadding: Dp = 10.dp
+    val ArtMakerControlMenuDropDownPadding: Dp = 12.dp
+    val CustomColorPalettePadding: Dp = 12.dp
+    val MenuItemSize: Dp = 32.dp
+    val ColorPickerMenuItemShapeSize: Dp = 32.dp
+    val ColorItemSize: Dp = 48.dp
+    val CustomColorPickerSize: Dp = 48.dp
+    val FullScreenDisabledPadding: Dp = 60.dp
+    val FullScreenEnabledPadding: Dp = 0.dp
+    val ArtMakerControlMenuHeight: Dp = 60.dp
+    val ArtMakerControlMenuShadowElevation: Dp = 60.dp
+    val CustomColorPaletteHeight: Dp = 330.dp
+    val ExportArtButtonSpacerHeight: Dp = 4.dp
+    val FullScreenToggleButtonSpacerHeight: Dp = 4.dp
+    val FirstColorsRowPadding: Dp = 4.dp
+    val FirstColorSetHorizontalArrangement: Dp = 4.dp
+    val FirstColorSetVerticalArrangement: Dp = 4.dp
+    val RecentColorsTextPadding: Dp = 4.dp
+    val SecondColorSetPadding: Dp = 4.dp
+    val SecondColorSetHorizontalArrangement: Dp = 4.dp
+    val SecondColorSetVerticalArrangement: Dp = 4.dp
+
+    // Missing UI Components' Dimensions...
```

Comment:
What do you mean missing UI? And could you please break these down so that they are grouped based on where they are used?

Reply:
The missing UI components are those that were not included in the `kmp-migration` branch such as `StrokePreview` and `LineStyleSelector`. Let me group the dimensions...